### ISSUE TITLE
No mentions to the original source code.

### ISSUE THREAD

**Issue by vdenotaris** [ulisesbocchio/spring-boot-security-saml-samples#13]
Dear Ulises,

You did an absolute good job in expanding and refactoring my project about a Spring Boot based SP.
However, it is sad to state that you just removed the mentions about all the contributions to the basic source code.

Sharing is caring, and *just by saying thanks*... Everybody wins.

**Comment by ulisesbocchio** [comment]
Sorry you feel that way @vdenotaris, I apologize for being so careless, where would you like me to add a mention to your project? What other contributors are you referring to?
It was definitely used as an inspiration for this sample: https://github.com/ulisesbocchio/spring-boot-security-saml-samples/tree/master/spring-security-saml-sample.
The spring boot plugin is merely based on that experience and my expertise on Spring Boot and Spring Security

**Comment by vdenotaris** [comment]
Hi. It is up to you find a right place for mentioning the original work: the README, the pom.xml... In my project there are several places where I mentioned all the people that helped me in making the solution better across the time. I actually used the Apache license with the sole purpose to make clear and visible the full "inspiration-path" as well in forks, as I did linking and mentioning Vladimir Schäfer's Spring SAML project.
